go.mod

```diff
@@ -78,7 +78,7 @@ require (
 	go.uber.org/zap v1.19.0
 	golang.org/x/crypto v0.1.0
 	golang.org/x/net v0.4.0
-	golang.org/x/oauth2 v0.0.0-20220223155221-ee480838109b
+	golang.org/x/oauth2 v0.3.0
```

Comment:
/hold

this bump gets us into the middle of the cloud/go/compute/metadata bump and triggers publishing bot breakage
cc @dims @nikhita 

xref https://github.com/kubernetes/kubernetes/issues/113366